16 Lazy Loading/webpack.config.js

```diff
@@ -14,16 +14,32 @@ module.exports = {
       extensions: ['', '.js', '.ts', '.tsx']
   },
 
-  entry: [
-    './index.tsx',
-    './css/site.css',
-    '../node_modules/toastr/build/toastr.css',
-    '../node_modules/bootstrap/dist/css/bootstrap.css'
-  ],
+  entry: {
+    app: './index.tsx',
+    styles: [
+      './css/site.css',
+      '../node_modules/toastr/build/toastr.css',
+      '../node_modules/bootstrap/dist/css/bootstrap.css'
```

Comment:
Sorry but I understand that the styles have to be embedded in app property? I miss something?